force-app/main/default/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -148,6 +186,45 @@ export default class rd2EntryForm extends LightningElement {
         return !this.isLoading && this.isSettingReady && this.isRecordReady ? "" : "slds-hide";
     }
 
+    renderedCallback() {
+        if (this.isExperienceSite && this.template.querySelector("lightning-record-edit-form")) {
```

Comment:
Can we put this block of code in its own function so we know why this is being called here?

Reply:
If this is working, we can keep it, but an alternative would be to just add a unique class to the form, then have all of the CSS begin with that new class name. That way, it can be included in the css file instead of added via Javascript.